test/specs/components/Button/Button-test.tsx

```diff
@@ -14,10 +20,91 @@ describe('Button', () => {
   isConformant(Button)
   buttonImplementsShorthandProp('icon', Icon, { mapsValueToProp: 'name' })
 
-  handlesAccessibility(Button, {
-    defaultRootRole: 'button',
-    accessibilityOverride: MenuBehavior,
-    overriddenRootRole: 'menu',
+  describe('accessibility', () => {
```

Comment:
tests for 'icon only' Button's accessibility features are missed - although those are described

Reply:
it seems that there is no generic logic that could ensure this fact, as `aria-label` (or its alternative versions) should be provided by client. The only thing that we could try here is to enforce mandatory coexistence of `iconOnly` prop and `aria` attribute in component props' TS types, which seems to be an overcomplex solution